projects/Mallard/src/screens/issue-screen.tsx

```diff
@@ -68,7 +68,6 @@ import {
 } from 'src/helpers/transform'
 import { FrontSpec } from './article-screen'
 import { useNavPositionChange } from 'src/hooks/use-nav-position'
-import { useConfig } from 'src/hooks/use-config'
```

Comment:
Can we keep everything in this file and relating to it? As its a separate optimisation that should have been done in another PR.

Reply:
👍